src/lwc/tileViewLwc/tileViewLwc.js

```diff
@@ -0,0 +1,20 @@
+import { LightningElement, api } from 'lwc';
+import getContactTile from '@salesforce/apex/ContactController.getContactTile';
+export default class tileViewLwc extends LightningElement {
+    contacts;
+    error;
+    connectedCallback() {
+        debugger;
+		this.loadContact();
+	}
+	loadContact() {
+		getContactTile()
```

Comment:
Fix indentation for this whole part. Use 4 spaces instead of 1 tab.

Reply:
Indentation fixed